site/src/api/queries/organizations.ts

```diff
@@ -0,0 +1,46 @@
+import type { QueryClient } from "react-query";
+import { API } from "api/api";
+import type {
+  CreateOrganizationRequest,
+  UpdateOrganizationRequest,
+} from "api/typesGenerated";
+import { meKey, myOrganizationsKey } from "./users";
+
+export const createOrganization = (queryClient: QueryClient) => {
+  return {
+    mutationFn: (params: CreateOrganizationRequest) =>
+      API.createOrganization(params),
+
+    onSuccess: async () => {
+      await queryClient.invalidateQueries(meKey);
+      await queryClient.invalidateQueries(myOrganizationsKey);
+    },
+  };
+};
+
+interface UpdateOrganizationVariables {
+  orgId: string;
+  req: UpdateOrganizationRequest;
+}
+
+export const updateOrganization = (queryClient: QueryClient) => {
+  return {
+    mutationFn: (variables: UpdateOrganizationVariables) =>
+      API.updateOrganization(variables.orgId, variables.req),
+
+    onSuccess: async () => {
+      await queryClient.invalidateQueries(myOrganizationsKey);
```

Comment:
In the other queries, the `meKey` is getting invalidated, why is it not here?

Reply:
because the `me` query only contains organization IDs.